index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="portfolio.css">
+    <title>Nsanzimfura portfolio</title>
+
+</head>
+<body>
+    <header>
+        <nav class="top-navigation-bar">
+            <h2 class="header-welcome">Welcome</h2>
+            <img class="menu-icon" src="./assets/header-files/icon-left-side-menu.svg" alt="menu icon" />
+        </nav>
+    </header>
```

Comment:
Nice work creating the nav bar, it looks perfect! 🏆 

- [ ] Good work including the `header` tag wrapping the `nav` tag, but this nav tag should only include the navigation links of your differents sections on the app, so wrap inside the nav tag only the navigation options (for now you only have the `img` of the menu, but later you will include differents sections of your portfolio)

It should look like this.

![image](https://user-images.githubusercontent.com/93630700/198167183-0f72267f-d196-47a6-8db6-6ef9b18931f1.png)

- [ ] Please add a display `flex` property to the `header`

- This gives us a lot of options to align all our elements, plus, it's responsive 🟢 🟢